### PR TITLE
[spark] Add Image Classification pyspark example

### PR DESCRIPTION
**Pull request by xyang16**

### CODE REVIEW

**Review by lanking520 — COMMENTED** [review]
Let's add a python jupyer notebook example on this (run in standalone mode)

**Comment by xyang16** [comment]
> Let's add a python jupyer notebook example on this (run in standalone mode)

OK

File: apache-spark/spark3.0/image-classification-pyspark/image_classification.py [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+    sc = spark.sparkContext
+    sc.setLogLevel("ERROR")
+
+    output_path = None
+    if len(sys.argv) > 1:
+        output_path = sys.argv[1]
+    else:
+        print("S3 output location not specified, printing results to output stream")
+
+    df = spark.read.format("image").option("dropInvalid", True).load("s3://alpha-djl-demos/temp/images/")
```
Comment (frankfliu) [inline apache-spark/spark3.0/image-classification-pyspark/image_classification.py:37]: s3://djl-ai/resources/demo/spark/image_classification

Comment (xyang16) [reply apache-spark/spark3.0/image-classification-pyspark/image_classification.py]: Done

**Review by lanking520 — APPROVED** [review]

**Pull request merged**